.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/test.md
```

Comment:
I see that in the .gitignore file node_modules/ and test.md is wrongly structured . In this way you will not push your node_modules/ and test.md file. Please change it as like following..  😄
node_modules/
test.md